templates/GridFieldLinkButton.ss

```diff
@@ -0,0 +1,5 @@
+<a class="action gridfield-button-link ss-ui-button ui-button ui-widget ui-state-default ui-corner-all ui-button-text-icon-primary"
+   href=$Link>
```

Comment:
Sorry, missed this earlier, please add quotes around this variable and ensure it is escaped - `"$Link.ATT"`, and `$Caption.XML` below